CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 ### Fixed
 
+- [#7084](https://github.com/thanos-io/thanos/pull/7084) Store: fix parameter validation for proxy store
```

Comment:
This is good but I think it's impossible to trigger this due to the check here https://github.com/thanos-io/thanos/blob/main/pkg/api/query/v1.go#L1034 so IMHO not worth mentioning it here.

Reply:
Ah yeah! I just noticed it in acceptance tests but not a user facing issue, yes!